pycryptobot.py

```diff
@@ -737,7 +738,8 @@ def executeJob(sc, market, granularity, tradingData=pd.DataFrame()):
             print ('   Buy Count :', buy_count)
             print ('  Sell Count :', sell_count, "\n")
 
-            print ('      Margin :', str(truncate((((sell_sum - buy_sum) / sell_sum) * 100), 2)) + '%', "\n")
+            margin_decimal = (sell_sum - buy_sum) / sell_sum if sell_sum else 0
```

Comment:
This just prevents potential DivisionByZeroError, which is what was happening.  I believe there is still the possibility that this number could be zero, or else I'd opt for leaving it the way it is for the sake of catching potential issues.

Reply:
Thanks, I think I fixed this already in the new major version I've been working on. The new version supports both Binance and Coinbase Pro. If you check out the "binance" branch you can see the new code. I am pretty sure I have a good fix for this so I will check how I did it and apply the update.

Reply:
This is how I did it in the "binance" version which resolved that DivisionByZeroError.

                if sell_count > 0:
                    print ('      Margin :', str(app.truncate((((sell_sum - buy_sum) / sell_sum) * 100), 2)) + '%', "\n")
                    
I'll use your update for this version. Thanks.